src/klein/py.typed

```diff
@@ -0,0 +1 @@
+# See: https://www.python.org/dev/peps/pep-0561/
```

Comment:
The actual content of this file seems weirdly underspecified. PEP 561 notes that it may contain `partial\n` and it is empty in the [example](https://github.com/ethanhs/sample-typed-package). This does seem to comply with the PEP, though.

Reply:
Huh, I hadn't noticed the `partial\n` thing (haven't made a stub package) but the PEP reads as the file existing being all that matters… unless you read that paragraph.

I'll keep the comment on the assumption that if it ever gets real syntax that `#` will mean comment… though as written, this PEP doesn't seem to really afford any compatibility with adding any syntax; if the comment had *ended* in `partial`, it should trigger the partial behavior… meh.